public/angular-app/settings/settings.html

```diff
@@ -0,0 +1,219 @@
+<div class="container">
+    <div class="row">
+        <div class="page-header">
+            <h1>Settings</h1>
+        </div>
+    </div>
+	<div class="row">
+        <form name="form" class="css-form" novalidate>
+           <div class="panel panel-default" style="margin-top:10px;">
+               <div class="panel-heading">
+                   <h3 class="panel-title">Project Info</h3>
+               </div>
+               <div class="panel-body">
+                   <form class ="form-horizontal">
+                       <fieldset class="form-group">
+                           <label class="control-label col-sm-3 text-right" style="margin-top: 8px;" for="httpAddress">
+                               Website Address
+                               <button class="fa-wrapper" ng-if="vm.settings.httpAddress">
+                                   <i class="fas fa-check fa-sm" style="color: green;"></i>
+                               </button>
+                               <button class="fa-wrapper" ng-if="!vm.settings.httpAddress">
+                                   <i class="fas fa-asterisk fa-sm" style="color: red;"></i>
+                               </button>
+                           </label>
+                           <div class="col-sm-9 no-padding-right">
+                               <input type="text"
+                                   name="link"
+                                   class="form-control"
+                                   id="httpAddress"
+                                   placeholder="www.missioncontrol.com"
+                                   ng-model="vm.settings.httpAddress" required>
+                               <div ng-show="form.$submitted || form.link.$touched">
+                                   <div ng-show="form.link.$error.required" class="text-danger">(*) Website Address is required.</div>
+                               </div>
+                           </div>
+                       </fieldset>
+                       <!-- <fieldset class="form-group">
```

Comment:
Can this whole commented out section be removed? It looks like it's from the project creation form.

Reply:
I think I removed that in the next PR.